lib/fixtures.ts

```diff
@@ -22,7 +22,7 @@ export function createFixtureLoader(overrideProvider?: providers.Web3Provider, o
       snapshot.id = await snapshot.provider.send('evm_snapshot', []);
       return snapshot.data;
     } else {
-      const provider = overrideProvider || createMockProvider();
+      const provider = overrideProvider || createGanacheProvider();
```

Comment:
We can't break backward compatibility. I would propose to have 3 functions:
```ts
createGanacheProvider() - creates ganache provider
createBuidlerProvider() - create Buidler provider
createMockProvider() - creates default provider (ganache by default, can be overridden in config)
```

Reply:
This is more or less how it works, but with one difference. To keep typing safe, `createMockProvider ` returns the same what `createGanacheProvider ` returns, with no additional options (but additionally warns that method is deprecated). In this particular line, we want to create Ganache provider, because Buidler does not support snapshots and reverts.

Reply:
Ok, if we don't deprecate `createMockProvider` yet, then we can keep using it in the code.